client/my-sites/email/email-management/home/email-plan-view.jsx

```diff
@@ -0,0 +1,198 @@
+import { CompactCard } from '@automattic/components';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import wp from 'calypso/lib/wp';
+import { hasEmailForwards } from 'calypso/lib/domains/email-forwarding';
+import { hasGSuiteWithUs } from 'calypso/lib/gsuite';
+import { hasTitanMailWithUs } from 'calypso/lib/titan';
+import SectionHeader from 'calypso/components/section-header';
+import HeaderCake from 'calypso/components/header-cake';
+import VerticalNav from 'calypso/components/vertical-nav';
+import VerticalNavItem from 'calypso/components/vertical-nav/item';
+import EmailTypeIcon from 'calypso/my-sites/email/email-management/home/email-type-icon';
+import MaterialIcon from 'calypso/components/material-icon';
+import Badge from 'calypso/components/badge';
+import getGSuiteUsers from 'calypso/state/selectors/get-gsuite-users';
+import { getEmailForwards } from 'calypso/state/selectors/get-email-forwards';
+import QueryEmailForwards from 'calypso/components/data/query-email-forwards';
+import { emailManagement } from 'calypso/my-sites/email/paths';
+
+class EmailPlanView extends React.Component {
+	state = {
+		isLoadingTitanMailboxes: false,
+		errorLoadingTitanMailboxes: false,
+		loadedTitanMailboxes: false,
+		titanMailboxes: [],
+	};
+
+	componentDidMount() {
+		this.loadTitanMailboxes();
+	}
+
+	componentDidUpdate() {
+		this.loadTitanMailboxes();
+	}
+
+	loadTitanMailboxes() {
+		const { selectedSite } = this.props;
+
+		if ( this.state.isLoadingTitanMailboxes || this.state.loadedTitanMailboxes ) {
+			return;
+		}
+
+		this.setState( {
+			isLoadingTitanMailboxes: true,
+		} );
+
+		wp.undocumented()
+			.getTitanMailboxesForSite( selectedSite.ID )
+			.then(
+				( data ) => {
+					this.setState( {
+						isLoadingTitanMailboxes: false,
+						errorLoadingTitanMailboxes: false,
+						loadedTitanMailboxes: true,
+						titanMailboxes: data?.accounts || [],
+					} );
+				},
+				() => {
+					this.setState( {
+						isLoadingTitanMailboxes: false,
+						errorLoadingTitanMailboxes: true,
+						loadedTitanMailboxes: true,
+						titanMailboxes: [],
+					} );
+				}
+			);
+	}
+
+	getEmails() {
+		const { domain, emails } = this.props;
+
+		if ( hasTitanMailWithUs( domain ) ) {
+			return this.state.titanMailboxes;
+		}
+
+		return emails;
+	}
+
+	renderEmailsList() {
+		const { translate } = this.props;
+
+		const emails = this.getEmails();
+
+		if ( ! emails || emails.length < 1 ) {
+			return;
+		}
+
+		const emailsItems = emails.map( ( email ) => {
+			return (
+				<CompactCard
+					key={ `email-row-${ email.email }` }
+					className="email-plan-view__mailbox-list-item"
+				>
+					<MaterialIcon icon="email" />
+					<span>{ email.email }</span>
+					{ email.isAdmin && <Badge type="info">{ translate( 'Admin' ) }</Badge> }
+				</CompactCard>
+			);
+		} );
+
+		return (
+			<div className="email-plan-view__mailbox-list">
+				<SectionHeader label={ translate( 'Mailboxes' ) } />
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Add New Mailboxes' )` *ES Score: 7*